tmt/template.py

```diff
@@ -0,0 +1,109 @@
+from typing import Any, Optional
+
+import tmt
+import tmt.utils
+from tmt.utils import Path, cached_property
+
+DEFAULT_CUSTOM_TEMPLATES_PATH = tmt.utils.CONFIG_PATH.expanduser() / 'templates'
+DEFAULT_PLAN_NAME = "/plans/default"
+INIT_TEMPLATES = ['mini', 'base', 'full']
+TEMPLATE_KEYS = ['story', 'plan', 'test', 'test_metadata']
```

Comment:
Maybe "types", or "kinds", or something similar? I mean, it's not just keys, they manifest as subdirectories, they correspond to well-established tmt objects like test or plan, so I'd go in this direction, if possible.

Reply:
Renamed in 78d7217caecb8c3bd05d92445264de9de4bfaf02.